.github/workflows/ci.yml

```
@@ -0,0 +1,19 @@
+name: GitHub Actions CI
+
+on: [push]
```

```suggestion
on: [push, pull_request]
```

This might be why the check didn't run (the branch is on your fork, so pushes there won't trigger the push event here)

---

Doesn't seem to have actually done it – if I'd done it in a branch, I believe it would have worked. I imagine this is a sort of spam protection on their end – if I automate opening a bajillion PRs to FOSS repos with CI, that could be a massive drain on their infra 😅

---

@bnb a good feedback for the actions team is to put "pull request" in as default.